src/ui/ui-kit/SurfaceList/SurfaceList.tsx

```diff
@@ -104,6 +104,7 @@ export const ItemButton = React.forwardRef<
           className={s.decoration}
           style={{
             border: outlined ? '1px solid var(--primary)' : undefined,
+            paddingBlock: 7,
```

Comment:
7? 🤨

Reply:
To avoid border overflow in virtualized lists :(

Reply:
Cause now this border renders out of item, makes its height 50 instead of 48